app/assets/stylesheets/layouts/dark.scss

```diff
@@ -1,81 +1,183 @@
+$font_color_subber: #DDDDDD;
+
+th, .content-header { background-color: $bg_color_head; color: $font_color_head; }
+.sub { background-color: $bg_color_sub; color: $font_color_sub; }
+.subber, .form-table-ender { background-color: $bg_color_subber; color: $font_color_subber; }
+
+/* - content */
+$bg_color_even: #3E3E3E;
+$bg_color_odd: #474747;
 
-$EVEN_COLOR: #3E3E3E;
-$ODD_COLOR: #474747;
 .even, .post-post, .post-reply:nth-child(even), #post-editor {
-  background-color: $EVEN_COLOR;
+  background-color: $bg_color_even;
 
   blockquote {
-    background-color: calculate-transparent-color($EVEN_COLOR, $ODD_COLOR, 0.2);
+    background-color: calculate-transparent-color($bg_color_even, $bg_color_odd, 0.2);
   }
 }
 .odd, .post-reply:nth-child(odd), .post-post + #post-editor, .post-post + a + #post-editor {
-  background-color: $ODD_COLOR;
+  background-color: $bg_color_odd;
 
   blockquote {
-    background-color: calculate-transparent-color($ODD_COLOR, $EVEN_COLOR, 0.2);
+    background-color: calculate-transparent-color($bg_color_odd, $bg_color_even, 0.2);
   }
 }
 
-#reply-icon-selector, .replace-icon, .icons-box, .icon-credit { background-color: #444D3F; }
-.subber, .flash.breadcrumbs, .form-table-ender {
-  background-color: #464646;
+
+$bg_color_post_description: #5D5D5D;
+//$font_color_post_description: #F3F3F3;
+$bg_color_navheader: #5F5D57;
+//$bg_color_navbutton: $colors_greybrown;
+
+.post-subheader { background-color: $bg_color_post_description; }
+.post-navheader { background-color: $bg_color_navheader; }
+
+// TODO: this doesn't seem to style #post-menu-box, is that ok?
+
+
+//$bg_color_icon_overlay: #000000; // box displaying current icon
+$bg_color_selectionpopup: #565656; // selection popups like character/alias on posts#editor
+$bg_color_expander: #6F7769; // expander on posts
+
+.selector-popup { background-color: $bg_color_selectionpopup; }
+.selector-popup-button { opacity: 0.7; } /* make the white less contrasty */
+.post-expander { background-color: $bg_color_expander; }
+
+$bg_color_post_icon: #444D3F;
+$bg_color_icon_selector: #444D3F;
+$font_color_icon_selector: #111111;
+
+.post-icon { background-color: $bg_color_post_icon; }
+#current-icon-dropdown { background-color: $bg_color_post_icon; color: $font_color_icon_selector; }
+#reply-icon-selector { background-color: $bg_color_icon_selector; }
+
+$bg_color_post_character: #595959;
+$bg_color_post_screenname: #515151;
+$bg_color_post_author: #4D4C4C;
+//$bg_color_post_character_narrow: $colors_lightbrown;
+//$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in // TODO: Should these flip on dark too?
+//$font_color_post_info: #F3F3F3;
+
+.post-character { background-color: $bg_color_post_character; }
+.post-screenname { background-color: $bg_color_post_screenname; }
+.post-author { background-color: $bg_color_post_author; }
+
+// TODO: This theme also seems seems not to style pagination
```

Comment:
leftover todo

Reply:
can we rephrase this to "TODO: Check pagination displays fine in this theme" and then I think it's safe to merge?